tests/conftest.py

```diff
@@ -0,0 +1,6 @@
+"""PyTest configuration."""
+
+
+def pytest_configure():
+    """Generate SQL files before running tests."""
+    exec(open("script/generate_sql").read())
```

Comment:
This probably shouldn't be called via exec, if only because it can behave differently than when called from a shell. Instead, we can use subprocess, similar to how we do [here](https://github.com/mozilla/gcp-ingestion/blob/24c1cea/ingestion-edge/tests/integration/conftest.py#L92)
```suggestion
    subprocess.call([sys.executable, "script/generate_sql"])
```

Reply:
alternately we should have a comment here explaining why `exec` is necessary, and a comment in the script explaining that it may be called via `exec`

Reply:
I'm running into issues here with paths (specifically `target/sql`) not being found if I use `subprocess.call()`.

Reply:
It looks acceptable to keep this as is here due to the complications of moving to `subprocess.call()`